yaks/lib/yaks/default_policy.rb

```diff
@@ -38,8 +38,13 @@ def derive_mapper_from_object(model)
       else
         klass = model.class
         begin
-          name = klass.name.split('::').last
-          return @options[:namespace].const_get(name + 'Mapper')
+          namespace, name = /(?:(.*)(?:::))?(.*)/.match(klass.name)[-2..-1]
```

Comment:
This line is very cryptic. Would it be possible to make it more explicit? maybe [extended regular expression syntax](http://tsigo.github.io/blog/2012/08/30/rubys-extended-regular-expressions/) would help?